proto/spire/common/common.proto

```diff
@@ -106,6 +109,7 @@ message RegistrationEntryMask {
     bool dns_names = 10;
     bool store_svid = 11;
     bool jwt_svid_ttl = 12;
+    bool hint = 14;
```

Comment:
why 14? may we use 13?

Reply:
Thanks for pointing this up, @MarcosDY 😄